.travis.yml

```diff
@@ -169,16 +169,16 @@ matrix:
         - bundle exec pod install
 
     - language: swift
-      name: "Swift - SPM - iOS (Xcode 11.2)"
+      name: "Swift - SPM - iOS (Xcode 11.1)"
```

Comment:
Why the downgrade?

Reply:
There is an issue with 11.2 that impacts SPM (it omits symbols in the compiled output when using generics in a few cases). It manifests as a crash. There are some compiler flags that I could add to work around it, but there's no real benefit to running CI on 11.2... so we can wait this one out until it's fixed in the next point release.